src/swarm-tui/Swarm/TUI/Controller.hs

```diff
@@ -327,50 +328,53 @@ handleMainEvent forceRedraw ev = do
     VtyEvent vev
       | isJust (s ^. playState . uiGameplay . uiDialogs . uiModal) -> handleModalEvent vev
     MouseDown (TerrainListItem pos) V.BLeft _ _ ->
-      playState . uiGameplay . uiWorldEditor . terrainList %= BL.listMoveTo pos
+      Brick.zoom playState $ uiGameplay . uiWorldEditor . terrainList %= BL.listMoveTo pos
```

Comment:
What's the benefit of using `Brick.zoom playState` for a single line of code like this?  It seems unnecessarily complicated to me.

Reply:
I suppose I was hoping to factor them out, but I'm not sure that will be possible.  Reverting them for now.